lib/src/base-styles/_nx-lists.scss

```diff
@@ -301,3 +301,38 @@
   margin-left: var(--nx-spacing-6x);
   max-width: var(--nx-width-text-wrapping);
 }
+
+.nx-list--numbered {
+  padding: 0 0 0 var(--nx-spacing-12x);
+
+  .nx-list__item {
+    list-style: decimal outside;
+    padding: var(--nx-spacing-2x) 0 0 0;
```

Comment:
Seems to be redundant

Reply:
> Seems to be redundant

https://github.com/sonatype/sonatype-react-shared-components/pull/616/commits/b907196c74622e842364d350d20bb69254016b12